tests/unit_tests/ensemble_evaluator/ensemble_evaluator_utils.py

```diff
@@ -194,11 +184,11 @@ def evaluate(self, config):
     def start(self):
         self._eval_thread.start()
 
-    def _shouldFailJob(self, real, step, job):
-        return (real, 0, step, job) in self.fail_jobs
+    def _shouldFailJob(self, real, job):
+        return (real, job) in self.fail_jobs
 
-    def addFailJob(self, real, step, job):
-        self.fail_jobs.append((real, 0, step, job))
+    def addFailJob(self, real, job):
```

Comment:
:scissors:  - not in use.

Reply:
https://github.com/equinor/ert/issues/6411